web/src/mobile/components/Orders/OrdersItem.tsx

```diff
@@ -41,29 +41,51 @@ const OrdersItemComponent = props => {
+                    <div className="pg-mobile-orders-item__row__block__value">
+                        <span style={{
+                            color: parseFloat(filled) < 100 ? 'var(--asks)' : 'var(--bids)'
+                        }}>
+                            {filled}%
+                        </span>
+                    </div>
                 </div>
                 <div className="pg-mobile-orders-item__row__block">
-                    <span>{intl.formatMessage({ id: 'page.mobile.orders.header.price' })}</span>
-                    <span className="pg-mobile-orders-item__row__block__value">
-                        <Decimal fixed={currentMarket.price_precision}>{actualPrice}</Decimal>
-                    </span>
+                    <div>
+                        <span>{intl.formatMessage({ id: 'page.mobile.orders.header.amount' })}</span>
+                        <span className="pg-mobile-orders-item__row__block__value">
+                            <Decimal fixed={currentMarket.amount_precision}>{order.origin_volume}</Decimal>
+                        </span>
+                    </div>
+                    <div className="pg-mobile-orders-item__second__row">
+                        <span>{intl.formatMessage({ id: 'page.mobile.orders.header.volume' })}</span>
+                        <span className="pg-mobile-orders-item__row__block__value">
+                            <Decimal fixed={currentMarket.price_precision}>{actualPrice}</Decimal>
+                        </span>
+                    </div>
                 </div>
                 <div className="pg-mobile-orders-item__row__block">
-                    <span>{intl.formatMessage({ id: 'page.mobile.orders.header.filled' })}</span>
-                    <div className="pg-mobile-orders-item__row__block__value">
-                        <span>{filled}%</span>
+                    <div>
+                        <span>{intl.formatMessage({ id: 'page.mobile.orders.header.price' })}</span>
+                        <span className="pg-mobile-orders-item__row__block__value">
+                            <Decimal fixed={currentMarket.price_precision}>{actualPrice}</Decimal>
+                        </span>
                     </div>
-                </div>
-                {order.state === 'wait' ? (
-                    <div className="pg-mobile-orders-item__row__button" onClick={props.handleCancel(order.id)}>
-                        <span>{intl.formatMessage({ id: 'page.mobile.orders.header.cancel' })}</span>
-                        <CloseIcon />
+                    <div className="pg-mobile-orders-item__second__row">
+                        <span>{intl.formatMessage({ id: 'page.mobile.orders.header.trigger' })}</span>
+                        <span className="pg-mobile-orders-item__row__block__value">
+                            <Decimal fixed={currentMarket.price_precision}>{actualPrice}</Decimal>
+                        </span>
```

Comment:
the trigger is Trigger price.  You can read about it in [architecture docs](https://github.com/openware/website/blob/main/internal/architecture/oip-010-advanced-orders.md#single-trigger)


Not all orders has trigger price value, that's why you need to check if `order. trigger_price ` exists, in case it doesn't - display `'-'`